include/class_loader/class_loader.hpp

```diff
@@ -132,9 +132,14 @@ class ClassLoader : public std::enable_shared_from_this<ClassLoader>
         std::bind(&ClassLoader::onPluginDeletion<Base>, shared_from_this(), std::placeholders::_1)
       );
     } catch (std::bad_weak_ptr & e) {  // This is not a shared_ptr
-      CONSOLE_BRIDGE_logWarn(
-        "class_loader::ClassLoader::createUniqueInstance "
-        "This class must be used with smart pointer");
+      static bool create_instance_message = false;
+      if (!create_instance_message) {
+        CONSOLE_BRIDGE_logWarn(
+          "To createUniqueInstance() with a class_loader::ClassLoader "
```

Comment:
@ahcorde oops, should be
```suggestion
          "To createInstance() with a class_loader::ClassLoader "
```